content/en/docs/release-oversight/risk-analysis.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Risk Analysis"
+description: Overview of Test Failure Risk Analysis.
+---
+
+### Summary
+In an effort to raise awareness for potential regressions, 
+tooling to analyze the overall risk for individual test failures has been created.  Risk 
+Analysis compares failed tests against the historical results of similar periodics and 
+ranks the 'Risk of Regression' based on the previous results. It **does not** analyze the code changed in a PR or payload. It 
+does consider the historical pass rate for failed tests 
+and provides a ranking based on that pass rate.  The motivation for the tooling is to help raise awareness 
+for test failures that historically are considered stable and are worth an additional review when failures are encountered.
+
+### Analyzing Risk of Test Failures
+Risk Analysis takes a closer look within jobs at the individual tests.  A 
+job that may fail more frequently than others can still provide signal for potential regression if a test, or tests, 
+that regularly succeed start to show failures.
+
+As e2e tests complete, origin based 
+[test failures are summarized](https://github.com/openshift/origin/blob/798a6eaf2327ecccf8f81c3a18127156aea72bd3/pkg/riskanalysis/write_test_failure_summary.go#L19) 
+for later evaluation. When 
+[openshift-e2e-test jobs complete](https://github.com/openshift/release/blob/fb6ba3c103e477a74bdf0d5faab5bb13cb1556af/ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh#L45) 
+the summaries are passed to 
+[Sippy for analysis](https://github.com/openshift/sippy/blob/376a0be3595f7f5ddda90e5f8834293ab93277e5/pkg/sippyserver/server.go#L1033).
+
+
+
+The test failures are submitted to Sippy along with metadata about the job itself.  That 
+data is used to find matching jobs within Sippy and analyze the historical pass rates, 
+typically over a 7 to 14 day period. The 
+higher the historical pass rate the higher the potential risk of regression.
+
+| Historical Pass Rate % | Risk Analysis Ranking |
+| :---: | :---: |
+| 100 - 98 | High |
+| 97 - 80 | Medium | 
+| < 80 | Low |
+
+
+<br><br>
+### Prow Job Artifacts ###
+Artifacts are created under the e2e test junit directory like `test-failures-summary_20230706-065159.json` and the final 
+analysis `test-risk-analysis.html`.  When available, the `test-risk-analysis.html` 
+summarization will show in the prow job spyglass view under the `Risk Analysis` 
+heading.  The view will show the Test Name, Risk Level, Risk Reason and 
+Open Bugs.  At the top of the table the Test Name `Overall` will 
+show the highest risk value.  The Open Bugs 
+are listed to surface any known issues with the test that may be impacting the results.  Additionally,
+ a link to Sippy is provided at the top of the section to aid in further investigation. 
```

Comment:
Can you clarify why the artifact is not always there.

Reply:
Not all jobs, like metal, run openshift-e2e-test-commands.  Mentioned in https://issues.redhat.com/browse/TRT-1167?focusedId=22710185&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-22710185

Also I think there are some failure cases like install fails and the junit directory isn't created then the artifacts can't be created.

Reply:
Are you wanting more info in the doc?  In the **Analyzing Risk of Test Failures** section I try to cover origin based test failures / openshift-e2e-test jobs.  I can try to outline that here to and discuss failures if need be.